### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by jdorfman**
Signed-off-by: Justin Dorfman <justindorfman@gmail.com>

### CODE REVIEW

**Comment by jdorfman** [comment]
This came from a conversation @tzuryby and I were having in Slack:

```
Tzury Bar Yochay Mar 22nd at 3:23 AM
Nice thread about the importance of README.md  and how to get it done right
https://news.ycombinator.com/item?id=26537805
```

```
Justin Dorfman  1 month ago
Yes ours can definitely use some love, it is on my todo list
```

File: README.md [hunk 1]
```diff
@@ -1,26 +1,58 @@
...
+
+### 🐳 Docker
+```bash
+git clone https://github.com/curiefense/curiefense.git
+cd curiefense/deploy/compose/
+docker-compose up
+```
+### ⛴ Helm
+```bash
+helm install curiefense
```
Comment (xavier-rbz) [inline README.md:28]: that doesn't work currently, right?

Comment (jdorfman) [reply README.md:28]: It's in the docs ¯\_(ツ)_/¯

Comment (jdorfman) [reply README.md]: Removed

File: README.md [hunk 2]
```diff
@@ -1,26 +1,55 @@
...
+docker-compose up
+```
+
+### 🐯 Katacoda
+This learning platform will allow you to explore what Curiefense can do without needing to install anything. Current lessons available:
+
+* [Deploy Curiefense](https://www.katacoda.com/curiefense/scenarios/getting-started)
+* [API Basics](https://www.katacoda.com/curiefense/scenarios/api-basics)
+
+## 📚 Documentation
```
Comment (flaper87) [inline README.md:33]: Can we have the Docs at the top? I love it when the README shows docs right there without me having to scroll anywhere

**Review by flaper87 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jdorfman) [reply README.md:33]: For sure

**Pull request merged**